compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/window/Dialog.kt

```diff
@@ -32,7 +32,8 @@ import androidx.compose.runtime.Immutable
 @Immutable
 expect class DialogProperties(
     dismissOnBackPress: Boolean = true,
-    dismissOnClickOutside: Boolean = true
+    dismissOnClickOutside: Boolean = true,
+    usePlatformDefaultWidth : Boolean = true,
```

Comment:
This cannot be in common (incompatible change with Android). Plus it's unrelated to PR topic anyway

Reply:
Oh, i guess it will be easier for you to commonize it from scratch. Should i close it? 
>This cannot be in common (incompatible change with Android).

What is the reason of such incompatibility? just interested.